_content/applications/tago/index.md

```diff
@@ -4,36 +4,47 @@ section: Integrations
 ---
 
-# Tago
+# TagoIO
 
-[Tago](https://tago.io/) allows you to quickly store, visualize, and act on sensor data.
+TagoIO is the best IoT platform to build nice dashboards, analytics, reports, and notifications easily and quickly. Also, you can manage devices and users, and add your own logo, colors, and url to deploy your solution in minutes.
 
-After creating an account on [Tago.io](https://tago.io/), activate the integration in The Things Network [Developer Console](https://console.thethingsnetwork.org/) by going to your application and Select Tago from the integration options.
+Here you will learn how to integrate TTN LoRaWAN devices with [TagoIO](https://tago.io/). You will be able to send data from your devices (uplink) to TagoIO and back to them (downlink).
 
-![Add-integration](Tago1.png)
 
-Think of a Process ID, select your access key and add (optional) the Authorization.
+## 1. TagoIO Setup
+After creating an account on [Tago.io](https://tago.io/), go to Devices, click on *'Add Devices'*, and filter by LoRaWAN TTN. Then, search for the device that you want to add. If you don't find it, select the type *Custom TTN Device*.
 
-> The **Authorization field** is not required, but it adds security to your integration, to use it, just type any secret text there and type the same code when generating the token at Tago
+> When selecting Custom TTN Device connector, a payload parse will NOT be included on the new device. You will need to add your own parser to extract values. 
 
-![Add-integration2](Tago2.png)
+Just give a name for the device, type the device EUI, and click on the **'Create Device'** button. 
 
+For example, if you selected the 'Custom TTN' type, you will be presented with the following screen. 
 
-## Uplink
+![Add-integration](Tago2.png)
 
-For your application to authenticate to Tago, you need to register your device at Tago.
+After the device is created, a standard parser script is included in the 'Payload Parser' tab based on the selected connector (unless you selected Custom TTN). Later, you can also add your own script to execute any calculation or to parse more data. 
 
-Go to the [Tago admin](https://admin.tago.io/), then go to **Device** and create a new one
+Then, you need to **create an Authorization** and use it in the TTN setup later. 
 
-![Uplink](Tago3.png)
+![Add-integration](Tago4.png)
 
-Go to tokens and type your *device EUI*.
+> This **Authorization** should be generated by TagoIO [here](https://admin.tago.io/devices/authorization); no Additional Parameters are needed
 
-> If you use the Authorization at TTN, type the same secret text in the Verification Code
+Copy this Authorization to use it in the next step.
 
-![Add-integration](Tago4.png)
+## 2. TTN Setup
+
+In The Things Network [Developer Console](https://console.thethingsnetwork.org/), go to your already created Application, select the 'Integrations' tab, and click **+add integration**.
```

Comment:
"The Things Network Console"

Reply:
Consistent highlighting of terms, quotes vs strong, please use **strong** for literal stuff to click on